### ISSUE TITLE
carto parser fails and visualization can't be loaded

### ISSUE THREAD

**Issue by javisantana** [CartoDB/cartodb#2845]
https://github.com/CartoDB/cartodb/blob/master/lib/assets/javascripts/cartodb/models/carto.js#L787

**Comment by javisantana** [comment]
(more info comming)

**Comment by javisantana** [comment]
the style that makes it fail:

```
/** category visualization */

#nycgaybarmaster {
   marker-fill-opacity: 1;
   marker-line-color: #FFF;
   marker-line-width: 0;
   marker-line-opacity: 1;
   marker-placement: point;
   marker-type: ellipse;
   marker-allow-overlap: true;
}

#nycgaybarmaster[open=true] {
  marker-fill-opacity: 1;
   marker-line-color: #FFF;
   marker-line-width: 0;
   marker-fill: #A53ED5;
}
#nycgaybarmaster[open=false] {
  marker-fill-opacity: .2;
   marker-line-color: #848480;
   marker-line-width: 2;
   marker-fill: #FFFFFF;
    [zoom = 12]{
    marker-width:3;
 		marker-line-width: .5;}
   
}
#nycgaybarmaster[open=null] {
   marker-fill: #B2DF8A;
}
#nycgaybarmaster 
	[zoom = 18]{
  		marker-width: 7;}
	[zoom = 17]{
  		marker-width: 7;}
	[zoom = 16]{
 		marker-width: 7;}
  	[zoom = 15]{
  		marker-width: 7;}
	[zoom = 14]{
 		marker-width: 7;}
    [zoom = 13]{
 		marker-width: 7;}
    [zoom = 12]{
 		marker-width: 4;}
    [zoom = 11]{
 		marker-width: 1;}


```

(yes, it's not cut)

**Comment by fdansv** [comment]
Solved in #2848
